index.html

```diff
@@ -41,6 +51,9 @@ <h2>Why Did It Have To Be Boxes...</h2>
                 <div class="box">Box 9</div>
                 <div class="box">Box 10</div>
             </div>
+
```

Comment:
Along with my comments in the CSS file about the boxes, you will need to do a little bit of work on each individual box to get everything to work.  It is also possible to not modify the code for the boxes and still get it to work properly, but you may need to do some research on CSS.